content/faq.md

```diff
@@ -65,6 +65,21 @@ If you want to switch back to dunst again, first define `SERVICE=org.knopwob.dun
 
 For more info, see the discussion in [#363](https://github.com/dunst-project/dunst/issues/363).
 
+## Cannot open a context menu with the keyboard shortcut
+
+There are some possible causes:
+
+* Unavailable keyboard shortcut. It is `ctrl+shift+period` by default, but possibly uneffective due to some reasons, e.g. unavailable keys on your keyboard layout or a conflicting shortcut with another application. Please try different shortcut via [the option `context` in the dunstrc](https://github.com/dunst-project/dunst/blob/732227eff5df7afa3f44bc7f2cc661b22b4e3f0b/dunstrc#L312).
```

Comment:
I want to avoid mentioning the keyboard shortcut functionality in the FAQ as it's [deprecated](https://github.com/dunst-project/dunst/blob/master/dunstrc#L287-L290), we should point users to `dunstctl` instead.

Reply:
Thanks, I'm planning to replace this topic totally. Because I don't know how to pull the change https://github.com/dunst-project/dunst-project.org/pull/17/commits/b9785204b51eaf39ed9228f37c2ee5af74cedf23 I'll recreate this PR...

Reply:
You can `git pull https://github.com/piroor/dunst-project.org/tree/add-faq-for-newcomers`

Reply:
Oh, I misunderstood that the commit is not related to my forked repository, sorry. Anyway I've already force-pushed the updated change.